randovania/gui/ui_files/preset_am2r_patches.ui

```diff
@@ -201,6 +201,20 @@
              </property>
             </widget>
            </item>
+           <item>
+            <widget class="QCheckBox" name="blue_save_doors_check">
+             <property name="text">
+              <string>Force blue Save Station Doors</string>
+             </property>
+            </widget>
+           </item>
+           <item>
+            <widget class="QLabel" name="blue_save_doors_label">
+             <property name="text">
+              <string>Ensures all Save Station doors are blue doors, even with door lock rando enabled.</string>
```

Comment:
I'm fine if you want to call it internally "Blue door" but AM2R has more than one door in the color of blue AFAIK.
Also it is called "Normal door" in the Door lock settings. If calling it blue door is not already ambiguous, it is at least questionable when calling it by a different name in another setting. 
The same applies to the changelog.

Reply:
whoops, yeah, that shouldnt happen. thanks.

Reply:
Altho I just noticed that Prime has the same issue xP
![grafik](https://github.com/randovania/randovania/assets/38186597/bbc2a408-655a-4f01-80d5-05aae5ad9489)
![grafik](https://github.com/randovania/randovania/assets/38186597/f7ec21dd-855e-4e90-a8d2-1ff8237783b5)
But that's up to change in a later PR i guess.